tests/src/lv_test_conf.h

```diff
@@ -15,14 +15,15 @@ extern "C" {
 /***********************
  * PLATFORM CONFIGS
  ***********************/
+#include "lv_test_malloc.h"
 
 #ifdef LVGL_CI_USING_SYS_HEAP
 #define LV_USE_BUILTIN_MALLOC   0
 #define LV_USE_BUILTIN_MEMCPY   1
 #define LV_USE_BUILTIN_SNPRINTF 1
 #define LV_STDLIB_INCLUDE <stdlib.h>
-#define LV_MALLOC       malloc
-#define LV_REALLOC      realloc
+#define LV_MALLOC       lv_test_malloc
+#define LV_REALLOC      lv_test_realloc
 #define LV_FREE         free
```

Comment:
Yes, we should have `lv_test_free` too. Who know what magic will `test_malloc` and `test_realloc` will do that should not be directly fed into `free`.

Reply:
Done in 5789edb